### ISSUE TITLE
Increase container horizontal padding

### ISSUE THREAD

**Issue by guywillis** [adaptlearning/adapt-contrib-vanilla#426]
### Subject of the enhancement 
Proposal to increase the container horizontal spacing from `.5rem` to `1rem` to allow for more breathing space between the content and the edge of the device.

### Your environment
* Master

**Comment by StuartNicholls** [comment]
@guywillis Does this mean the margins around the content container?

**Comment by oliverfoster** [comment]
The gutter down each side of the content on a phone screen, it's too thin at the moment.

**Comment by StuartNicholls** [comment]
@guywillis / @oliverfoster I've created a pull request #435 that addresses the horizontal space between the gutters & the margins, So desktop layout, increase to the gutters (the space between columns) and the minimum margins (the space either size of the content) in mobile view.

**Comment by guywillis** [comment]
> @guywillis Does this mean the margins around the content container?

Yes, the horizontal container padding attached to elements like blocks and notify
